plexapi/client.py

```diff
@@ -621,7 +621,7 @@ def _loadData(self, data):
         self.protocol = data.attrib.get('protocol')
         self.providerIdentifier = data.attrib.get('providerIdentifier')
         self.ratingKey = utils.cast(int, data.attrib.get('ratingKey'))
-        self.repeat = utils.cast(bool, data.attrib.get('repeat'))
+        self.repeat = data.attrib.get('repeat')
```

Comment:
Shouldn't this be casted to an `int`?
```suggestion
        self.repeat = utils.cast(int, data.attrib.get('repeat'))
```

Reply:
I'm too old for this typed language stuff. :)